pages/home.py

```diff
@@ -16,6 +16,8 @@ class HomePage(Base):
     _displayed_profile_name_locator = (By.CSS_SELECTOR, '.billboard h3')
     _task_in_progress_locator = (By.ID, 'task-in-progress')
     _pick_a_task_locator = (By.ID, 'pick-a-task')
+    _displayed_suggested_first_tasks_list_locator = (By.CSS_SELECTOR, '.task-list-container h3')
+    _suggested_first_tasks_list_locator = (By.CSS_SELECTOR, '.task-list > li')
```

Comment:
This locator will find each individual task, so we probably shouldn't call it a `list_locator`. I think just `task_locator` would suffice, as this is the locator to one individual task.